src/App.tsx

```diff
@@ -141,26 +142,31 @@ const App = () => {
         });
     };
 
-    const toggleCurrentPage = (link: String) => (link === 'About' ? setIsProduct(false) : setIsProduct(true));
-
     return (
         <div className="body">
             <PagesContext.Provider value={{ page: numberPage, maxPages: totalPages }}>
-                <HeaderComponent theme={theme} amountCart={amountCart} toggleCurrentPage={toggleCurrentPage} changeTheme={changeTheme} />
-                {!isProduct && <AboutComponent theme={theme} />}
-                {isProduct && (
-                    <ProductList
-                        theme={theme}
-                        products={products}
-                        increasePage={increasePage}
-                        decrementPage={decrementPage}
-                        increaseCounter={increaseCounter}
-                        decrementCounter={decrementCounter}
-                        getProductsByCategory={getProductsByCategory}
-                        getSortByName={getSortByName}
-                    />
-                )}
-                <FooterComponent theme={theme} />
+                <Routes>
+                    <Route path={'/'} element={<LayoutComponent theme={theme} amountCart={amountCart} changeTheme={changeTheme} />}>
+                        <Route path={'/'} index element={<About theme={theme} />} />
+                        <Route
+                            path={'/products'}
+                            element={
+                                <ProductList
+                                    theme={theme}
+                                    products={products}
+                                    increasePage={increasePage}
+                                    decrementPage={decrementPage}
+                                    increaseCounter={increaseCounter}
+                                    decrementCounter={decrementCounter}
+                                    getProductsByCategory={getProductsByCategory}
+                                    getSortByName={getSortByName}
+                                />
+                            }
+                        />
+                        <Route path={'/product/:id'} element={<ProductPageComponent theme={theme} />} />{' '}
```

Comment:
path={'product/:id'} is it work without `/`?

Reply:
fixed